toolchain/crosstool-ng/crosstool.config

```diff
@@ -326,16 +335,20 @@ CT_LINUX_V_5_4=y
 # CT_LINUX_V_3_10 is not set
 # CT_LINUX_V_3_4 is not set
 # CT_LINUX_V_3_2 is not set
-CT_LINUX_VERSION="5.4.70"
+CT_LINUX_VERSION="5.4.248"
```

Comment:
This is change made by `./ct-ng arm-remarkable-linux-gnueabihf`. I guess it doesn't matter, as we only need kernel headers, which should not change in minor releases (not sure about that though)

Reply:
We would prefer to target the exact kernel version in use on the device.

Reply:
Have you found in the documentation why this is the case? Or opened an issue about it?

I'm not exactly happy that it's ignoring our configuration.